permafrost/views.py

```diff
@@ -204,12 +211,19 @@ def get_form_kwargs(self):
                 kwargs['site'] = self.request.site
         return kwargs
 
+
 # List Permission Groups
 class PermafrostRoleListView(PermafrostSiteMixin, FilterByRequestSiteQuerysetMixin, ListView):
     model = PermafrostRole
     queryset = PermafrostRole.on_site.all()
     permission_required = ["permafrost.view_permafrostrole"]
+
+    def get_queryset(self):
+        qs = super(PermafrostRoleListView, self).get_queryset()
+        # Should be reflected in TC
+        return qs.filter(name__in=PERMAFROST_DEFAULT_ROLES)
```

Comment:
`PERMAFROST_DEFAULT_ROLES` will be updated to
 [
        'Student',
        'Administrator',
        'Curriculum Designer',
        'Instructor',
        'Site Owner'
]

Reply:
change to exclude list instead of default list to allow users to create new roles

Reply:
good catch